kpops/cli/pipeline_config.py

```diff
@@ -30,7 +30,8 @@ class PipelineConfig(BaseSettings):
     """
 
     defaults_path: Path = Field(
-        default=...,
+        default=Path("."),
+        example="defaults",
```

Comment:
We should add in the description what the default value is. In other words, what is the behaviour if the defaults_path is not set